contracts/ERC721M.sol

```diff
@@ -491,6 +509,15 @@ contract ERC721M is IERC721M, ERC721AQueryable, Ownable, ReentrancyGuard {
         emit Withdraw(value);
     }
 
+    /**
+     * @dev Withdraws ERC-20 funds by owner.
+     */
+    function withdrawERC20(address mintCurrencyWithdrawn) external onlyOwner {
```

Comment:
thoughts on allowing a recipient address to be passed here?

Reply:
@nothing0012 is there a reason there isn't an address param in the native currency `withdraw` function? I followed the convention there

If no reason, happy to add the param to this one

Reply:
i think its not required, but could be nice if we want to distribute funds directly from the contract to the project since it can save us an extra transfer

Reply:
Think either way there's an extra transfer, since the project would have to send us our share so they'd have to do the transfer

Medium-term the plan is actually to consider inheriting from the PaymentSplitter so that ME and the Creator can each just get their own share without having to go through the other